codespell_lib/data/dictionary.txt

```diff
@@ -25863,6 +25863,7 @@ sintax->syntax
 Sionist->Zionist
 Sionists->Zionists
 siply->simply
+sircular->circular
```

Comment:
Add sircle too?

Reply:
Yes, indeed. Done in 155dadf4aa0395998634ac74d94231ca1315ca85 (for all others as well).

Thanks a lot for the suggestions :+1:

Reply:
sircles too perhaps?

Reply:
Yes, indeed. Added in 35fbb1e52237e23ea6e110d0c3ab352e3246c8f2